lib/tasks/certs.rake

```diff
@@ -202,4 +204,57 @@ namespace :certs do
       end
     end
   end
+
+
+
+  desc 'Check that LG certificate bundle matches certificates in certificate path'
+  task check_certificate_bundle: :environment do |t, args|
+    CertificateStore.instance.load_certs!(dir: 'config/certs')
+    bundled_certs = []
+    cert_bundle_file = File.read(IdentityConfig.store.login_certificate_bundle_file)
+    cert_bundle = cert_bundle_file.split(CertificateStore::END_CERTIFICATE).map do |cert|
+      cert += CertificateStore::END_CERTIFICATE
+      cert = Certificate.new(OpenSSL::X509::Certificate.new(cert))
```

Comment:
`cert` is the local name inside this block, so reassigning it doesn't do anything, right? since its value is returned in the `.map` that's the same as this right?

```suggestion
      Certificate.new(OpenSSL::X509::Certificate.new(cert))
```

Reply:
yep, thank you